ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/intended/structured_configs/DC1-SVC3A.yml

```diff
@@ -730,7 +730,10 @@ port_channel_interfaces:
   shutdown: false
   mode: trunk
   vlans: 110-111,210-211
-  mlag: 10
```

Comment:
Hmm do we expect to "lose" `mlag` here? it could be because DC1-SVC4A is not a real device but not sure)